sanity_tests.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+#
+GROUP_VARS_FILE=group_vars/all.yml
+
+function do_we_have_ip_addresses {
+   for node in haproxy-0 haproxy-1 master-0 worker-0; do
+       grep ${node} ${GROUP_VARS_FILE} | grep -E "([0-9]{1,3}\.){3}[0-9]{1,3}" > /dev/null
+       if [[ $? -ne 0 ]]; then
+           echo "ERROR: ${node} does not have a valid IP address."
+           echo "(This can be caused if you have another stack in your tenancy, stealing your IP Addresses.)"
```

Comment:
This is a reference to a specific heat deployment issue - bear in mind this is supposed to be separate from the heat code, so whilst its valid to check the correct config is in place, if this is deployed on another cloud provider or in vCenter for a non cloud deployment then this comment isn't relevant and could be considered confusing. It'd be better to stick to what should be there and why its needed and let the person deploying troubleshoot why.